### ISSUE TITLE
[Requested feature] Skript support

### ISSUE THREAD

**Issue by Rickard-Martensson** [AJCStriker/Movecraft-3#165]
Will there ever be Skript support for the plugin. For those of you who done know, Skript is a plugin that allows you to make plugin inside it. Thanks!

**Comment by mwkaicz** [comment]
I think that you can script Movecraft by using Skript and CBWrapper combination just right now. 
(CBWrapper is plugin that creates dummy player ghost which can call commands like a player on specific location)

**Comment by Rickard-Martensson** [comment]
That helps with some things, thanks :D

But other stuff like detect where a craft is, or force a craft to move is not possible. thanks anyways

**Comment by mwkaicz** [comment]
With calling CBWrapper you can create "virtual" dummy player and sudo command thru him.
With each call of CBWrapper you set location of the player, so it's like you remote controll some player.
and if you know the start point, where you set the command, the direction and speed of the craft, you can suspect ,where is it. ... or you can use some sort of testForBLock commandBlock ... testFor entity doesn't works for CBW.

If you want, I can show you some live example

**Comment by Rickard-Martensson** [comment]
Yeah, that would be awesome. 
are you refering to this video?: https://www.youtube.com/watch?v=1w_rem0_fKY

**Comment by mwkaicz** [comment]
Not exactly, this video is deprecated and pretty much deprecated since version 1.9. At this point you dont need neither repeater. I'm building small example at gcmc.cz:25902 now, if you have free time now, come. Or let me know here, when you can come another day.

**Comment by mwkaicz** [comment]
@MrPancake sorry, my internet connection failed. If you have some questions, you can find me almost everyday between 21:00-24:00 on TS3 server gcmc.cz

**Comment by mwkaicz** [comment]
@MrPancake I made last changes in redstones, updated map is on same url.
